pkgs/development/python-modules/mahotas/default.nix

```
@@ -1,24 +1,29 @@
-{ buildPythonPackage, fetchFromGitHub, nose, pillow, scipy, numpy, imread, lib, stdenv }:
+{ buildPythonPackage, fetchFromGitHub, pillow, scipy, numpy, pytestCheckHook, imread, freeimage, lib, stdenv }:
 
 buildPythonPackage rec {
   pname = "mahotas";
-  version = "1.4.10";
+  version = "1.4.11";
 
   src = fetchFromGitHub {
     owner = "luispedro";
     repo = "mahotas";
     rev = "v${version}";
-    sha256 = "0fjiyl82wj1a6xzr9mss2y2rydl4zchl2cbdbg0jm0fcrs99q4hw";
+    sha256 = "029gvy1fb855pvxvy8zwj44k4s7qpqi0161bg5wldfiprrysn1kw";
   };
 
   # remove this as soon as https://github.com/luispedro/mahotas/issues/97 is fixed
   patches = [ ./disable-impure-tests.patch ];
 
-  propagatedBuildInputs = [ numpy imread pillow scipy ];
-  checkInputs = [ nose ];
+  propagatedBuildInputs = [ numpy imread pillow scipy freeimage ];
+  checkInputs = [ pytestCheckHook ];
```

could remove patch
```suggestion
  checkInputs = [ pytestCheckHook ];
  
  disabledTests = [
    "test_ellipse_axes"
    "test_normalize"
    "test_haralick3d"
  ];
```
and please add comment as to why they need to be disabled

---

updated with your suggestions